prdoc/pr_3979.prdoc

```diff
@@ -0,0 +1,21 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Deprecate `para_id()` from `CoreState` in polkadot primitives
+
+doc:
+  - audience: "Node Dev"
+    description: |
+     The motivation is that the old implementation of `para_id()` can be misused with Coretime which allows two
```

Comment:
Given that we just fixed it, it can not actually be misused anymore. So the only reason for deprecation is that claim queue is replacing it.